src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/AlertAddressForm.tsx

```diff
@@ -5,36 +5,71 @@
 "use client";
 
 import {
+  AriaCheckboxProps,
   AriaRadioProps,
+  mergeProps,
+  useCheckbox,
   useFocusRing,
   useRadio,
   useRadioGroup,
 } from "react-aria";
-import { RadioGroupState, useRadioGroupState } from "react-stately";
+import {
+  RadioGroupState,
+  useRadioGroupState,
+  useToggleState,
+} from "react-stately";
 import styles from "./AlertAddressForm.module.scss";
 import { useL10n } from "../../../../../../hooks/l10n";
-import { createContext, useContext, useRef } from "react";
-import type { EmailUpdateCommOptionRequest } from "../../../../../../api/v1/user/update-comm-option/route";
+import { createContext, useContext, useRef, useState } from "react";
+import type {
+  EmailUpdateCommOptionRequest,
+  EmailUpdateCommTypeOfOptions,
+} from "../../../../../../api/v1/user/update-comm-option/route";
 import { VisuallyHidden } from "../../../../../../components/server/VisuallyHidden";
 import { useSession } from "next-auth/react";
-
-export type AlertAddress = "primary" | "affected";
+import { FeatureFlagName } from "../../../../../../../db/tables/featureFlags";
 
 export type Props = {
-  defaultSelected: AlertAddress;
+  breachAlertsEmailsAllowed: boolean | null;
+  monitorReport: boolean | null;
```

Comment:
Nit: we're not passing the report itself, but whether the user has enabled it:

(And reminder that if you rename via F2 or Right Click->Rename Symbol, it will update references to it as well.)

```suggestion
  monitorReportAllowed: boolean | null;
```